packages/ContentAutotagging/tsconfig.json

```diff
@@ -23,5 +23,5 @@
     "baseUrl": "."
   },
   "exclude": ["node_modules", "./dist"],
-  "include": ["./src/**/*.ts", "./*/src/**/*.ts"]
+  "include": ["./src/**/*.ts", "./*/**/*.ts"]
```

Comment:
Doesn't really hurt, but `"./*/**/*.ts"` includes `"./src/**/*.ts"` already, so having both is redundant